src/main/java/com/marek/onlinebookstore/service/order/OrderService.java

```diff
@@ -0,0 +1,20 @@
+package com.marek.onlinebookstore.service.order;
+
+import com.marek.onlinebookstore.dto.order.OrderDto;
+import com.marek.onlinebookstore.dto.order.OrderItemDto;
+import com.marek.onlinebookstore.dto.order.PlacingOrderRequestDto;
+import com.marek.onlinebookstore.model.User;
+import java.util.List;
+import org.springframework.data.domain.Pageable;
+
+public interface OrderService {
+    OrderDto makeOrder(PlacingOrderRequestDto requestDto);
+
+    List<OrderDto> getHistory(User user, Pageable pageable);
```

Comment:
The method 'getHistory' should be provided with a user identifier rather than an entire user object. This would make the method easier to use and would only require the necessary information.